src/Simplify.elm

```diff
@@ -5373,10 +5373,11 @@ basicsToFloatChecks =
         )
 
 
-intToIntChecks : IntoFnCheck
-intToIntChecks =
+intToIntChecks : (Float -> Int) -> IntoFnCheck
+intToIntChecks operation =
```

Comment:
I guess this should be renamed, maybe something like `toIntChecks` or `conversionToIntChecks`?